core/src/main/codegen/templates/Parser.jj

```diff
@@ -2969,26 +2971,62 @@ SqlNodeList OrderBy(boolean accept) :
             throw SqlUtil.newContextException(s.pos(), RESOURCE.illegalOrderBy());
         }
     }
-    <BY> AddOrderItem(list)
-    (
-        // NOTE jvs 6-Feb-2004:  See comments at top of file for why
-        // hint is necessary here.
-        LOOKAHEAD(2) <COMMA> AddOrderItem(list)
-    )*
+    <BY> OrderItemList(list)
+    {
+        return new SqlNodeList(list, s.addAll(list).pos());
+    }
+}
+
+/**
+ * Parses a BY clause for SELECT (syntactic sugar for GROUP BY ... ORDER BY).
+ */
+SqlNodeList SqlSelectBy() :
```

Comment:
This is non-standard, so it may cause problems by accepting programs that others think are illegal.
So the question is whether it should be behind a configuration flag, like the * EXCEPT () feature you have introduced recently.

Reply:
Should I also make this feature supported only in Babel?